src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/MarketplaceCustomisation.tsx

```diff
@@ -0,0 +1,62 @@
+import { t, Trans } from '@lingui/macro'
+import { Form, FormInstance, Input } from 'antd'
+import { MinimalCollapse } from 'components/MinimalCollapse'
+import TooltipIcon from 'components/TooltipIcon'
+import { useAppSelector } from 'hooks/AppSelector'
+
+import { MarketplaceFormFields } from '.'
+
+export function MarketplaceCustomisation({
+  form,
+}: {
+  form: FormInstance<MarketplaceFormFields>
+}) {
+  const {
+    nftRewards: {
+      collectionMetadata: { name, description, symbol },
+    },
+  } = useAppSelector(state => state.editingV2Project)
+
+  const initialValues = {
+    collectionName: name,
+    collectionDescription: description,
+    collectionSymbol: symbol,
+  }
+  return (
+    <MinimalCollapse
```

Comment:
**suggestion**: to make this component simpler, WDYT about:
- moving this MinimalCollapse to the parent component
- making _this_ component return a `Form`
- renaming _this_ component to `NftMarketplaceCustomizationForm`

Reply:
Done